dix/req.h

```diff
@@ -283,7 +285,8 @@ struct m0_dix_next_reply {
 /** Initialises DIX request. */
 M0_INTERNAL void m0_dix_req_init(struct m0_dix_req  *req,
 				 struct m0_dix_cli  *cli,
-				 struct m0_sm_group *grp);
+				 struct m0_sm_group *grp,
+				 uint32_t min_replicas);
```

Comment:
If we decide to use _dr_min_success_ instead of _dr_min_replicas_ then please consider names in functions API.